### PR TITLE
Update German from Transifex

### PR DESCRIPTION
**Pull request by Gootector**
Update German from Transifex

Greetings,
Gootector

Resolves: *(direct link to the issue)*

*(short description of the changes and the motivation to make the changes)*

- [x] I signed [CLA](https://www.audacityteam.org/cla/)
- [x] The title of the pull request describes an issue it addresses
- [x] If changes are extensive, then there is a sequence of easily reviewable commits
- [x] Each commit's message describes its purpose and effects
- [x] There are no behavior changes unnecessary for the stated purpose of the PR

Recommended:
- [x] Each commit compiles and runs on my machine without known undesirable changes of behavior

### CODE REVIEW

File: locale/de.po [hunk 1]
```diff
@@ -4465,7 +4465,7 @@ msgid ""
 "\n"
 "You are saving directly to a slow external storage device\n"
 msgstr ""
-"Aufgezeichneter Ton ging an den gekennzeichneten Stellen verloren. Mögliche Ursachen:\n"
+"Aufgezeichneter Audio ging an den gekennzeichneten Stellen verloren. Mögliche Ursachen:\n"
```
Comment (LWinterberg) [inline locale/de.po:4468]: Aufgezeichnetes Audio...

File: locale/de.po [hunk 2]
```diff
@@ -11759,11 +11759,11 @@ msgstr "Exportierte Tags"
 
 #: src/export/Export.cpp
 msgid "All selected audio is muted."
-msgstr "Sämtlicher ausgewählter Ton ist stummgestellt."
+msgstr "Sämtlicher ausgewählter Audio ist stummgestellt."
 
 #: src/export/Export.cpp src/export/ExportMultiple.cpp
 msgid "All audio is muted."
-msgstr "Sämtlicher Ton ist stummgestellt."
+msgstr "Sämtlicher Audio ist stummgestellt."
```
Comment (LWinterberg) [inline locale/de.po:11766]: Sämtliches Audio

**Review by LWinterberg — CHANGES_REQUESTED** [review]

**Review by LWinterberg — APPROVED** [review]

**Comment by Gootector** [comment]
@LWinterberg Two strings with dots at the end. See: https://github.com/audacity/audacity/pull/5366.

**Comment by Gootector** [comment]
Typo - my mistake, sorry.

**Pull request merged**